delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/context/RemoteContextResolution.scala

```diff
@@ -47,6 +47,18 @@ trait RemoteContextResolution {
       case ContextRemoteIri(uri) => Set(uri)
       case _                     => Set.empty
     }
+
+  /**
+    * Merges the current [[RemoteContextResolution]] with the passed ones
+    */
+  def merge(others: RemoteContextResolution*): RemoteContextResolution =
+    new RemoteContextResolution {
+      override def resolve(iri: Iri): Result[Json] = {
+        others.map(_.resolve(iri)).toList.foldLeft(self.resolve(iri)) { (acc, c) =>
```

Comment:
It would be better to use foldLeftM which could provide short-circuiting when we succeed